docs/standard/commandline/snippets/define-commands/csharp/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿// <all>
+using System.CommandLine;
+
+class Program
+{
+    static async Task Main(string[] args)
+    {
+        var rootCommand = new RootCommand("Sample command-line app");
+
+        rootCommand.SetHandler(() =>
+        {
+            Console.WriteLine(rootCommand.Description);
```

Comment:
Would the intent be more clear if this was "Hello, World". I do not think users would ever output the description.